### DISCUSSION TITLE
Which one is the text repunctuation and recapitalization model?

### DISCUSSION THREAD

**Discussion by yxd886** [snakers4/silero-models#176] [Q&A] [answered]
Hello, I am looking for an onnx  model for  repunctuation and recapitalization, since the readme has said that you have published a model for this, could you please tell me which one is for this purpose?  Thanks very much!

**Comment by snakers4** [comment, marked as answer]
Hi,

https://github.com/snakers4/silero-models#text-enhancement

**Reply by yxd886** [reply]
Thanks very much!  However, I need an onnx format model, and I tried to convert this to onnx but failed. Could you please provide an onnx format model?

**Reply by snakers4** [reply]
I am afraid this is not possible now
The model contains internal logic

**Reply by yxd886** [reply]
Ok, thanks any way
